src/test/scala/pl/allegro/tech/marathon/plugin/auth/FakeHttpResponse.scala

```diff
@@ -0,0 +1,22 @@
+package pl.allegro.tech.marathon.plugin.auth
+
+import mesosphere.marathon.plugin.http.HttpResponse
+
+class FakeHttpResponse(
```

Comment:
You can't know purpose of this class without its context and I would inline fake object construction to specific test case.